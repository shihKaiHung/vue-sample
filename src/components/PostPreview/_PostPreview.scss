.PostPreview {
    // Vars

    // Support

    // Module
    & {
        //
    }

    // Facets

    // States
}
