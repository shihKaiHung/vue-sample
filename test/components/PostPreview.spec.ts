import "jest";
import { mount } from "@vue/test-utils";
import PostPreview from "@components/PostPreview/PostPreview.vue";

describe("PostPreview", () => {
  test("is a Vue instance", () => {
    const wrapper = mount(PostPreview, {
      propsData: {
        post: {
          id: 1,
          title: "hello",
          description: "test"
        }
      }
    });
    expect(wrapper).toMatchSnapshot();
  });
});
