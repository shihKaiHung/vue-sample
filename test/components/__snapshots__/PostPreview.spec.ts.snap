// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostPreview is a Vue instance 1`] = `
<div>
  <h2>hello</h2>
  <p>test</p>
</div>
`;
